composer.json

```diff
@@ -1,43 +1,45 @@
 {
-    "name": "dnj/laravel-ticketing",
-    "description": "Package used for the ticketing system.",
-    "type": "library",
-    "require": {
-        "php": "^8.1"
-    },
-    "require-dev": {
-        "phpunit/phpunit": "^9.5",
-        "phpstan/phpstan": "^1.4.1",
-        "friendsofphp/php-cs-fixer": "^3.1",
-        "orchestra/testbench": "^7.0"
-    },
-    "autoload": {
-		"psr-4": {
-			"dnj\\Ticket\\": "src/"
-		}
-	},
-	"autoload-dev": {
-		"psr-4": {
-			"dnj\\Ticket\\Tests\\": "tests/"
-		}
-	},
-	"scripts": {
-		"test:types": "vendor/bin/phpstan analyze",
-		"test:phpunit": "vendor/bin/phpunit",
-		"test:codestyle": "vendor/bin/php-cs-fixer fix -v --dry-run --stop-on-violation --using-cache=no",
-		"test": [
-			"@test:types",
-			"@test:phpunit",
-			"@test:codestyle"
-		]
-	},
-	"extra": {
-		"laravel": {
-			"providers": [
-				"dnj\\Ticket\\TicketServiceProvider"
-			]
-		}
-	},
-    "license": "MIT",
-    "minimum-stability": "dev"
+  "name": "dnj/laravel-ticketing",
+  "description": "Package used for the ticketing system.",
+  "type": "library",
+  "require": {
+    "php": "^8.1"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "^9.5",
+    "phpstan/phpstan": "^1.4.1",
+    "friendsofphp/php-cs-fixer": "^3.1",
+    "orchestra/testbench": "^7.0",
+    "laravel/legacy-factories": "^1.0.4"
```

Comment:
What is this? can you tell me why we need this?

Reply:
When using the factory model without this package, I received an error.
Currently, I changed the factory build method in the new commit and this package is no longer needed.
I also removed it from the composer.json file